Mathlib/LinearAlgebra/Alternating/Curry.lean

```diff
@@ -0,0 +1,84 @@
+/-
+Copyright (c) 2022 Eric Wieser. All rights reserved.
```

Comment:
For others to verify: these lemmas were added in https://github.com/leanprover-community/mathlib3/pull/14802, so I agree with this header.